passwordless_test.go

```diff
@@ -35,32 +36,33 @@ func (g testGenerator) Sanitize(ctx context.Context, s string) (string, error) {
 }
 
 func TestPasswordless(t *testing.T) {
+	ctx := context.TODO()
```

Comment:
Tests should use the background context.
```suggestion
	ctx := context.Background()
```